app/mailers/mailer.rb

```diff
@@ -42,7 +42,7 @@ def notifiers_changed(user_id)
 
   def gem_pushed(pushed_by_user_id, version_id)
     @version = Version.find(version_id)
-    owners_to_notify = @version.rubygem.owners.where(ownerships: { notifier: true }).pluck(:email)
+    owners_to_notify = @version.rubygem.owners.with_ok_inbox.where(ownerships: { notifier: true }).pluck(:email)
```

Comment:
I was hoping we use a callback in mailer. We will need this logic for ownership and adoption requests as well.
https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-callbacks

Reply:
Are ownership and adoption requests on `master` yet? If yes, where can I find them. If not, can we go with the simplest implementation for now?

Reply:
No they are not. If you really want to stick with this then sure.. we can change it later.